### PR TITLE
Fixes `return None` in CodeWriter.

### PR DESCRIPTION
**Pull request by sighingnow**

### CODE REVIEW

**Comment by sighingnow** [comment]
The CI failure is not cuased by this PR.

File: Cython/Tests/TestCodeWriter.py [hunk 1]
```diff
@@ -107,6 +107,18 @@ def test_cascaded_assignment(self):
     def test_attribute(self):
         self.t(u"a.x")
 
+    def test_return_none(self):
+        self.t(u"""
+                    def f(x, y, z):
+                        return
+                    cdef f(x, y, z):
+                        return
+                    def f(x, y, z):
```
Comment (da-woods) [inline Cython/Tests/TestCodeWriter.py:116]: I think it should work fine, but can we have a `return None` test in here too?

Comment (sighingnow) [reply Cython/Tests/TestCodeWriter.py:116]: Now such cases has been covered as well.

**Comment by sighingnow** [comment]
@da-woods could we get this PR merged ?

**Comment by scoder** [comment]
Thanks

**Pull request merged**